tools/sidekick/library/library.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, viewport-fit=cover"
+    />
+    <meta name="Description" content="AEM Sidekick Library" />
+    <meta name="robots" content="noindex" />
+    <base href="/" />
+
+    <style>
+      html,
+      body {
+        margin: 0;
+        padding: 0;
+        font-family: sans-serif;
+        background-color: #ededed;
+        height: 100%;
+      }
+
+      helix-sidekick { display: none }
+    </style>
+    <title>Sidekick Library</title>
+  </head>
+
+  <body>
+    <script
+      type="module"
+      src="https://www.hlx.live/tools/sidekick/library/index.js"
+    ></script>
+    <script>
+      const library = document.createElement('sidekick-library')
+      library.config = {
+        base: '/block-library/library.json',
+        autoblocks: '/tools/sidekick/library/plugins/autoblocks/autoblocks.js',
+        templates: '/tools/sidekick/library/plugins/templates/templates.js',
+        placeholders: '/tools/sidekick/plugins/placeholders/placeholders.js'
```

Comment:
is there a reason you added this, don't think we had this plugin originally. copy/pasta error perhaps?

Reply:
Ah, yea must have been a copy/paste mistake. My bad, will remove the `placeholders` plugin.